release-notes/7.0/preview/api-diff/preview1/Microsoft.NETCore.App/7.0-preview1_System.IO.md

```diff
@@ -0,0 +1,14 @@
+# System.IO
+
+``` diff
+ namespace System.IO {
+     public sealed class FileInfo : FileSystemInfo {
+-        public override string ToString();
```

Comment:
@carlossanlop It seems that `FileInfo` was never overriding `ToString`, but it's base type (`FileSystemInfo`) does. So we have not removed this method, but most likely the tool used to generate the API diff was updated?

Reply:
The change is correct. It was made in this PR: https://github.com/dotnet/runtime/pull/63139

The comparison shows that the API indeed used to exist in 6.0, and is now gone:
![image](https://user-images.githubusercontent.com/1175054/154791063-5a32e5c8-b09d-455d-9836-38da427ada0b.png)